app/uk/gov/hmrc/plasticpackagingtax/registration/views/check_liability_details_answers_page.scala.html

```diff
@@ -134,6 +135,31 @@
                                 )
                             )
                         ))
+                    ),
+                    SummaryListRow(
+                        key = Key(
+                            content = Text(messages("checkLiabilityDetailsAnswers.membersUnderGroupControl"))
+                        ),
+                        value = Value(
+                            registration.groupDetail match {
+                                case Some(group) =>
+                                  group.membersUnderGroupControl match {
+                                      case Some(true) =>Text(messages("general.true"))
+                                      case Some(false) => Text(messages("general.false"))
+                                      case _ => Empty
+                                  }
+                                case _ => Empty
+                            }
+                        ),
+                        actions = Some(Actions(
+                            items = Seq(
+                                ActionItem(
+                                    href = pptRoutes.RegistrationTypeController.displayPage().url,
```

Comment:
Is this intentional (linking to the registration type page rather than the group under same control page)?

Reply:
Changed to under group control page